### PR TITLE
Use container-based infrastructure for Travis CI

### PR DESCRIPTION
**Pull request by tricknotes**
Travis CI recommends to migrate infrastructure.
See more details: http://docs.travis-ci.com/user/migrating-from-legacy/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 language: ruby
+sudu: false
```
Comment (hibariya) [inline .travis.yml]: :eyes: sudo ?

Comment (tricknotes) [inline .travis.yml]: Yes! :innocent:

Comment (kakutani) [inline .travis.yml]: :eyes: sudu?

Comment (tricknotes) [inline .travis.yml]: Right :innocent:

**Comment by tricknotes** [comment]
We need to use `apt-get install` via `sudo`.
Thought `sudo: false` is irrelevance here...

**Comment by tricknotes** [comment]
> We need to use `apt-get install` via `sudo`.

Using `addons.apt.packages` key resolves this issue: https://travis-ci.org/idobata/idobata-hooks#L86
ref:  http://docs.travis-ci.com/user/migrating-from-legacy/#Adding-APT-Packages

**Comment by hibariya** [comment]
Wow, good to know :smiley:

**Comment by hibariya** [comment]
@tricknotes Thanks!

**Pull request merged**